src/rpc_client/src/compute_client.rs

```diff
@@ -160,6 +160,14 @@ impl RpcClient for ComputeClient {
     async fn new_client(host_addr: HostAddr) -> Result<Self> {
         Self::new(host_addr).await
     }
+
+    async fn new_clients(host_addr: HostAddr, size: usize) -> Result<Vec<Self>> {
```

Comment:
Can we make this a default implementation of the trait?

Reply:
Sure.